crates/ui/src/input/input.rs

```diff
@@ -85,6 +85,7 @@ actions!(
         MoveToPreviousWord,
         MoveToNextWord,
         TextChanged,
+        Clean
```

Comment:
This action is better rename to `Escape` like `Enter`, because in the future, Esc will have some other functions.

And here you have missed some details, for example we selected something, if press `Escape`, it must cancel selection first.